app.js

```diff
@@ -231,6 +315,77 @@ function checkNodes() {
 setInterval(checkNodes, 60*1000);
 checkNodes();
 
+function getAccounts() {
+  // gets accounts from core and stores it in redis
+  axios.get(MONITOR_URL+'accounts').then(function(resp){
+    var day = moment();
+    let multi = redisClient.multi();
+    var key = "accounts_"+day.format("YYYY-MM-DD")
+    multi.set(key, resp.data.count);
+    // Expire the key after 90 days
+    multi.expire(key, 60*60*24*90);
+
+    multi.exec(function (err, res) {
+      if (err) {
+        console.error(err);
+        return;
+      }
+      console.log("Added total accounts stats: "+key+" "+res);
+    });
+  });
+}
+setInterval(getAccounts, 3600000);
+getAccounts();
+
+
+function getAssets() {
+  // gets assets from core and stores it in redis
+  axios.get(MONITOR_URL+'assets').then(function(resp){
+    var day = moment();
+    let multi = redisClient.multi();
+    var key = "assets_"+day.format("YYYY-MM-DD")
+
+    multi.set(key, resp.data.count);
+    // Expire the key after 90 days
+    multi.expire(key, 60*60*24*90);
+
+    multi.exec(function (err, res) {
+      if (err) {
+        console.error(err);
+        return;
+      }
+      console.log("Added total assets stats: "+key+" "+res);
+    });
+  });
+}
+setInterval(getAssets, 3600000);
+getAssets();
+
+function getDexVolume() {
+  // gets assets from core and stores it in redis
+  axios.get('http://ticker.stellar.org').then(function(resp){
+    var day = moment();
+    let multi = redisClient.multi();
+    var key = "dex_"+day.format("YYYY-MM-DD")
+
+    for (var i = 0; i < resp.data.length; i++) {
+      multi.hmset(key, resp.data[i].Name, JSON.stringify(resp.data[i]));
+    }
+    // Expire the key after 90 days
+    multi.expire(key, 60*60*24*90);
+
+    multi.exec(function (err, res) {
+      if (err) {
+        console.error(err);
+        return;
+      }
+      console.log("Added DEX volume stats: "+key+" "+res);
+    });
+  });
+}
+setInterval(getDexVolume, 3600000);
```

Comment:
I think we should call it more often. If it gets called at `11:01pm` we will lose information about all accounts created/deleted in the last hour of that day. I think calling this every 5 minutes or even every minute shouldn't be a problem for a `core-mon` database.